src/styles/colors.ts

```diff
@@ -12,6 +12,7 @@ const color = {
     blueSky: '#9ECCEA',
     blueSkyDark: '#51A5DA',
     blueSkyLight: '#E8EAED',
+    culturedBlue: '#EBEFF9',
```

Comment:
could this be better DX (developer experience if it was BlueCultured? The reason is when i type `color.bl` i am hoping to see all the blues listed by intellisence in my code editor. This shade of blue will be hidden unless someone opens the file and checks it manually